apollo-runtime/src/main/java/com/apollographql/android/impl/RealApolloCall.java

```diff
@@ -74,7 +76,7 @@ private RealApolloCall(Operation operation, HttpUrl serverUrl, Call.Factory http
     return executeNetworkRequest();
   }
 
-  @Nonnull @Override public ApolloCall<T> enqueue(@Nullable final Callback<T> callback) {
+  @Nonnull @Override public void enqueue(@Nullable final Callback<T> callback) {
```

Comment:
We need to guarantee that the a Call has been executed when we watch. Otherwise we have no dependentKeys to monitor. So I don't like relying on have the user chain `apollocall.enqueue.watch()`. Instead I have it has `apolloCall.toWatcher().enqueueAndWatch()`.

Is there a value in having `enqueue` return an ApolloCall? OkHttp has void return null. It just seems dangerous. Once you call `enqueue` many functions on that `ApolloCall` now explode, and returning it seems to imply that the object is still safe to use.